smap/sources/conf/aqi_converter_conf.ini

```diff
@@ -0,0 +1,14 @@
+[server]
+Port = 8005
+
+[/]
+uuid = 2aae1481-14b2-13e1-1a44-3703cd4c5e01
```

Comment:
This is the UUID for this driver. For UUID of sensors, I am fetching them in setup using API already built.